include/swift/Basic/CBasicBridging.h

```diff
@@ -132,31 +67,31 @@ _Bool JSON_value_getAsNull(void *valuePtr);
 _Bool JSON_value_getAsBoolean(void *valuePtr, _Bool *result);
 _Bool JSON_value_getAsString(void *valuePtr, BridgedData *result);
 _Bool JSON_value_getAsDouble(void *valuePtr, double *result);
-_Bool JSON_value_getAsInteger(void *valuePtr, long long *result);
+_Bool JSON_value_getAsInteger(void *valuePtr, int64_t *result);
 _Bool JSON_value_getAsObject(void *valuePtr, void *_Nullable *_Nonnull result);
 _Bool JSON_value_getAsArray(void *valuePtr, void *_Nullable *_Nonnull result);
 
-unsigned long JSON_object_getSize(void *objectPtr);
-BridgedData JSON_object_getKey(void *objectPtr, unsigned long i);
+size_t JSON_object_getSize(void *objectPtr);
+BridgedData JSON_object_getKey(void *objectPtr, size_t i);
 _Bool JSON_object_hasKey(void *objectPtr, const char *key);
 void *JSON_object_getValue(void *objectPtr, const char *key);
 
-long long JSON_array_getSize(void *arrayPtr);
-void *JSON_array_getValue(void *arrayPtr, long long index);
+size_t JSON_array_getSize(void *arrayPtr);
+void *JSON_array_getValue(void *arrayPtr, size_t index);
```

Comment:
Should this be `int64_t` or was `long long` wrong?

Reply:
This is a index of the array element, since the argument of the `operator[]` is `size_t`, I thought we should use it.

Reply:
I really meant both here - ie. both the `getSize` and the `getValue`. If it's the right thing, sure. I mostly just went through and checked for any inconsistencies.